knexfile.js

```diff
@@ -0,0 +1,48 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'pg',
+    connection: {
+      host: 'localhost',
+      database: 'ticketdb',
+      user: 'postgres',
+      password: 'testPass',
```

Comment:
We can store things like the password in a .ENV file (i.e by using the package dot-env) to hide the password from malicious actors who (regularly) scrape github for data like this.